presto-docs/src/main/sphinx/installation/deployment.rst

```diff
@@ -172,6 +172,110 @@ These properties require some explanation:
   the host and port of the Presto coordinator. This URI must not end
   in a slash.
 
+* ``query.queue-config-file``:
+  Specifies the file to read the queue configuration from. The queue configuration
+  is described in more detail below.
+
+Queue Configuration
+^^^^^^^^^^^^^^^^^^^
+
+The queueing rules are defined in a json file, and control the number of queries
+that can be submitted to Presto and the quota of running queries per queue.
+Rules that specify multiple queues will cause the query to enter the queues sequentially.
+Rules are processed sequentially, and the first one that matches will be used.
+In the example configuration below, there are five queue templates. In the
+"user.${USER}" queue "${USER}" will be expanded to the name of the user
+that submitted the query. "${SOURCE}" is also supported, which expands to the
+source submitting the query.
+
+There are also five rules that define which queries go into which queues.
+
+#. This rule states that "bob" is an admin
```

Comment:
It seems more accurate to say "this rule matches when" rather than "this rule states"